dts/arm/atmel/samx7x.dtsi

```diff
@@ -0,0 +1,482 @@
+/*
+ * Copyright (c) 2024 Henrik Brix Andersen <henrik@brixandersen.dk>
```

Comment:
Add after 2017

Reply:
Thanks. I've removed the line instead, as I opted to submit my clean-up of that file in a follow-up PR in order to keep this PR as small as possible.